contrib/inventory/consul_io.py

```diff
@@ -189,8 +190,8 @@ def setup_logging():
 try:
   import consul
 except ImportError as e:
-  print("""failed=True msg='python-consul required for this module. see
-  http://python-consul.readthedocs.org/en/latest/#installation'""")
+  sys.stderr.write("""failed=True msg='python-consul required for this module. see
+  http://python-consul.readthedocs.org/en/latest/#installation'\n""")
   sys.exit(1)
 
```

Comment:
wouldn't it be better to?:

```sys.exit("""failed=True msg='python-consul required for this module. see
 +  http://python-consul.readthedocs.org/en/latest/#installation'\n"""```

^ will exit with rc=1 and print to stderr in py2/py3 compatible fashion.